iroha_2_whitepaper.md

```diff
@@ -47,7 +46,7 @@ We use Hyperledger Ursa.
 
 ### 2.4. Smart Contracts
 
-Smart contracts (chaincode) are supported using a WASM sandbox and the entire smart contract lifecycle is supported. If a smart contract is deployed within a channel, only channel participants can see it and invoke it.
+Iroha provides a library of smart contracts called **I**roha **S**pecial **I**nstructions (ISI).
```

Comment:
do we give up term "Commands" in favour of ISI?

Reply:
Yes, we call them smart contracts now and not commands.